_includes/manuals/3.16/1.2_release_notes.md

```diff
@@ -4,16 +4,222 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### User Interface & Experience
+- **All Hosts Page Redesign** - Continued modernization of the All Hosts page with enhanced functionality, eliminating AngularJS pages and improving user experience for managing thousands of hosts
```

Comment:
@jeremylenz @parthaa

Reply:
I dont think we need to mention about eliminating Angular JS part. Rest seems ok to me.